trunk/SUAVE/Components/Energy/Networks/Internal_Combustion_Propeller.py

```diff
@@ -0,0 +1,153 @@
+## @ingroup Components-Energy-Networks
+# Internal_Combustion_Propeller.py
+# 
+# Created:  Sep 2016, E. Botero
+# Modified: Apr 2018, M. Clarke 
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+# suave imports
+import SUAVE
+
+# package imports
+import numpy as np
+from SUAVE.Components.Propulsors.Propulsor import Propulsor
+from SUAVE.Core import Data, Units
+
+# ----------------------------------------------------------------------
+#  Network
+# ----------------------------------------------------------------------
+## @ingroup Components-Energy-Networks
+class Internal_Combustion_Propeller(Propulsor):
```

Comment:
not quite sure what this entails?  what makes this different than a regular prop?  maybe a more clear description or name?